{
  "uuid": "b4c08573-efdc-48bb-a75c-df8db45e40ae",
  "path": "/usr/lib/python3.12/lib2to3/fixes/fix_execfile.py",
  "size": 2048,
  "modified": "2024-04-12T13:14:55Z",
  "created": "2025-05-28T17:29:43.455611334Z",
  "sha256_hash": "6ff65db1192099457cb3d9f2618a893c6ac430028550284f3a34d5c08042b0eb",
  "file_type": "Script",
  "permissions": 33188,
  "is_executable": true,
  "is_signed": false,
  "code_signature": null,
  "bundle_info": null,
  "scan_timestamp": "2025-06-01T18:46:04.921712613Z"
}